inc/checks/class-php-in-upload.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace runcommand\Doctor\Checks;
+
+use WP_CLI;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+
+/**
+ * Errors when new WordPress minor release is available; warns for major release.
+ */
+class PHP_In_Upload extends Check {
+
+	/**
+	 * Array containing list of files found in the uploads folder
+	 *
+	 * @var array
+	 */
+	protected $php_files_array = array();
+
+	public function run() {
+
+		// Path to the uploads folder.
+		$wp_content_dir = defined( 'WP_CONTENT_DIR' ) ? WP_CONTENT_DIR . '/uploads/' : ABSPATH . 'wp-content/uploads/';
```

Comment:
This is not the correct way of detecting the uploads folder. There's a function `wp_upload_dir()` you can use. Amongst others, it checks for an `UPLOADS` constant that overrides the default uploads path.
See: https://developer.wordpress.org/reference/functions/wp_upload_dir/